Specifications/Events.Store.MongoDB/Events/for_EventConverter/when_converting_to_runtime_stream_event/a_non_aggregate/event_log_event.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.Runtime.Events.Store.Streams;
+using Machine.Specifications;
+
+namespace Dolittle.Runtime.Events.Store.MongoDB.Events.for_EventConverter.when_converting_to_runtime_stream_event.a_non_aggregate
+{
+    public class event_log_event
+    {
+        static Event stored_event;
+        static IEventConverter event_converter;
+        static Runtime.Events.Store.Streams.StreamEvent result;
+
+        Establish context = () =>
+        {
+            stored_event = events.an_event_not_from_aggregate(0);
+            event_converter = new EventConverter();
+        };
+
+        Because of = () => result = event_converter.ToRuntimeStreamEvent(stored_event);
+
+        It should_return_a_committed_event = () => result.Event.ShouldBeOfExactType<CommittedEvent>();
+        It should_have_the_correct_committed_event = () => result.Event.ShouldBeTheSameAs(stored_event);
+        It should_have_the_correct_stream_position = () => result.Position.Value.ShouldEqual(stored_event.EventLogSequenceNumber);
```

Comment:
This looks off - `should_have_the_correct_STREAM_POSITION` => `Position.Value.ShouldEqual(stored_event.EVENTLOGSEQUENCENUMBER)`

Reply:
I disagree, it makes sense given the context "when converting to runtime stream event a non aggregate event log event". The StreamPosition of that StreamEvent should be equal to the EventLogSequenceNumber of the stored event

Reply:
That makes sense, I guess I didn't really understand what ToEventLogEvent was doing